e2e-tests/numpy-test.yaml

```diff
@@ -8,11 +8,16 @@ pipeline:
 
 # Test python/import, and python/test pipelines
 test:
+  environment:
+    # TODO(pnasrat): fix to use multiple python
+    contents:
+      packages:
+        - python-3.12
```

Comment:
You should not need this, and it might actually just complicate debugging of a future error case.

Its nice to be explicit, but I think this will cause issue in teh future when numpy moves to 3.13 (default version) and then python-3.12 will be installed along side of python-3.13-base, the 'import' below will just give an error like

```
$ python3.12 -c 'import numpy'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'numpy'
```